examples/EFD_data_analysis/CCW-Rotator-EFD.ipynb

```diff
@@ -0,0 +1,236 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DM-25824: Rotator/CCW synchronization test\n",
+    "\n",
+    "This notebook contains queries to the EFD to analyse the data from the CCW-Rotator synchronization test.\n",
+    "\n",
+    "The test consists of performing repetitive movements with the Rotator and make sure that the CCW can keep up.\n",
+    "\n",
+    "The test can be driven by a notebook (`CCW-Rotator-IT.ipynb`) or using a Script (`maintel/integration_tests/ccw_rotator_stress.py` in `ts_standardscripts`)."
```

Comment:
I will confess it makes me very nervous to have two different ways to do the same thing. It's hard to believe that both will be maintained.